<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotControlWindow</class>
 <widget class="QMainWindow" name="RobotControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Robot Control Panel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_robotInfo">
       <layout class="QFormLayout" name="formLayout_robotInfo">
        <item row="1" column="0">
         <widget class="QLabel" name="label_name">
          <property name="text">
           <string>Robot Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_robotName"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_type">
          <property name="text">
           <string>Robot Type</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBox_robotType">
          <item>
           <property name="text">
            <string>GroundBot</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DroneBot</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QPushButton" name="btn_createRobot">
          <property name="text">
           <string>Create Robot</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Speed</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinBox"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Miission active</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Altitude</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="spinBox_2"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Mode robot</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_dashboard">
       <layout class="QVBoxLayout" name="layout_dashboard">
        <item>
         <widget class="QLabel" name="label_robotNameDashboard">
          <property name="text">
           <string>Robot: --</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_status">
          <property name="text">
           <string>Status: Idle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_startMission">
          <property name="text">
           <string>Start Mission</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_emergencyStop">
          <property name="text">
           <string>Emergency Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_settings">
       <layout class="QFormLayout" name="layout_settings">
        <item row="0" column="0">
         <widget class="QLabel" name="label_speed">
          <property name="text">
           <string>Speed (0â€“100)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_speed">
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_mode">
          <property name="text">
           <string>Mode</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBox_mode">
          <item>
           <property name="text">
            <string>Patrol</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Search</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Return</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_altitude">
          <property name="text">
           <string>Altitude</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBox_altitude">
          <property name="maximum">
           <number>500</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="btn_applySettings">
          <property name="text">
           <string>Apply Settings</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="navButtons">
      <item>
       <widget class="QPushButton" name="btn_goToInfo">
        <property name="text">
         <string>Robot Info</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_goToDashboard">
        <property name="text">
         <string>Dashboard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_goToSettings">
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
